config/locales/en.yml

```diff
@@ -57,8 +57,8 @@ en:
   investigation: sequence
   activity: activity
   interactive: interactive
-  material: material
-  instructional_materials: Instructional Materials
+  material: assignment
+  instructional_materials: Assignments
```

Comment:
There are several usages of `t('material')` and most of these should not be changed to assignment.
Also there a few usages of `t('instructional_materials')` only of one of these should be changed to Assignments.

You should either replace the `t('*')` usage with Assignments directly in the places that should be changed. Or you could continue to use i8n, and add a new key 'assignment: assignment', and then you can use pluralize and capitalize to get it in the right form in the various places.

If you need help figuring out what are the 'right places', let me know.